indico/modules/events/papers/operations.py

```diff
@@ -302,6 +321,15 @@ def create_comment(paper, text, visibility, user):
     comment = PaperReviewComment(user=user, text=text, visibility=visibility)
     paper.last_revision.comments.append(comment)
     db.session.flush()
+    receivers = {x for x in paper.cfp.judges}
+    if visibility == PaperCommentVisibility.contributors or visibility == PaperCommentVisibility.reviewers:
+        receivers |= paper.cfp.layout_reviewers if paper.cfp.layout_reviewing_enabled else {}
+        receivers |= paper.cfp.content_reviewers if paper.cfp.content_reviewing_enabled else {}
+    if visibility == PaperCommentVisibility.contributors:
+        receivers |= {x.person.user for x in paper.contribution.person_links}
```

Comment:
a person may not have an indico user associated so you should check for this - but I think it'd actually be better to use the person.email (which may also be empty, so take care of that!) here.